### PR TITLE
use "set outfit" window for /looktype

### PR DESCRIPTION
**Pull request by Zbizu**
### Pull Request Prelude

- [x] I have followed [proper The Forgotten Server code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [X] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed
/looktype now sends outfit window
/looktype param now sends outfit window with a single outfit
obraz
obraz

[cont]: https://github.com/otland/forgottenserver/wiki/Contributing
[code]: https://github.com/otland/forgottenserver/wiki/TFS-Coding-Style-Guide

### CODE REVIEW

**Comment by Zbizu** [comment]
small note: this change targets 12.9, will crash earlier clients because it sends outfits up to gnomprona

**Comment by soul4soul** [comment]
Cool change but it should only send the outfit window when an outfit id isn't passed. I have used this for quick testing of outfit ids and it would be annoying to get that dialog all the time.

File: data/talkactions/scripts/looktype.lua [hunk 1]
```diff
@@ -1,4 +1,3 @@
--- keep it ordered
```
Comment (ranisalt) [inline data/talkactions/scripts/looktype.lua:1]: Doesn't need to keep _sorted_ anymore?

**Comment by omarcopires** [comment]
This doesn't work, it just crashes the client.

**Pull request closed without merging**